lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -538,7 +538,7 @@ def billing_name(payment_source, options)
           elsif options[:billing_address] && options[:billing_address][:name].present?
             options[:billing_address][:name]
           end
-        name ? I18n.transliterate(name || '')[0..29] : nil
+        name ? I18n.transliterate(name).gsub(/[^a-zA-Z ]/, ' ').strip[0..29] : nil
```

Comment:
Certain special characters doesn't cause issues. I wonder if there's a different way to look for emojis